### DISCUSSION TITLE
AGPL3 to GPLv2+ Relicense Consent

### DISCUSSION THREAD

**Discussion by Nyeriah** [azerothcore/azerothcore-wotlk#19869] [General]
If you have contributed to AzerothCore in the past 5 years please write below your explicit consent on the commits you authored being relicensed to GPLv2+.

Note: this only includes contributions to https://github.com/azerothcore/azerothcore-wotlk
Modules have their own license and remain unaffected by this

NOTE: The consent must be EXPLICIT in the message you leave here.

All off-topic messages will be removed.

**Comment by BeefSteaker** [comment]
I give full consent for the Relicensing of all my contributions to Azerothcore

**Comment by Nyeriah** [comment]
I give full consent for the relicensing of all my past contributions to AzerothCore to GPLv2+

**Comment by cdenq** [comment]
I give full consent for the Relicensing of all my contributions to Azerothcore

**Comment by noisiver** [comment]
I give full consent for the Relicensing of all my contributions to Azerothcore

**Comment by r4d1sh** [comment]
I give full consent for the Relicensing of all my contributions to Azerothcore

**Comment by stoudtlr** [comment]
I give full consent for the relicensing of all my past contributions to AzerothCore to GPLv2+

**Comment by SoulSeekkor** [comment]
I give full consent for the relicensing of all my past contributions to AzerothCore to GPLv2+

**Comment by heyitsbench** [comment]
I give consent for my contributions past and future to be relicensed from AGPLv3 to GPLv2 or later regarding both AzerothCore and its modules.

**Comment by razers211** [comment]
I give full consent for the Relicensing of all my contributions to Azerothcore

**Comment by Day36512** [comment]
I give full consent for the relicensing of all my contributions to Azerothcore

**Comment by crilleaz** [comment]
I give full consent for the Relicensing of all my contributions to Azerothcore

**Comment by Gozzim** [comment]
I give full consent for the Relicensing of all my contributions to Azerothcore

**Comment by Tereneckla** [comment]
I give full consent for the relicensing of all my past and future contributions to AzerothCore and the modules under the AcerothCore Organisation to GPLv2+

**Comment by M4th3m4tic4l** [comment]
I authorize all my small contributions to be licensed under the GPLv2 license :) (in this account and in other accounts I had)

**Comment by Bogir** [comment]
I give full consent for the relicensing of all my past contributions to AzerothCore to GPLv2+

**Comment by Nelnamara** [comment]
I give my full consent for relicensing of any contributions I have made to the project.

**Comment by Helias** [comment]
I give full consent for the relicensing of all my past contributions to AzerothCore to GPLv2+.

**Comment by squeakyneb** [comment]
(Does this include the wiki and/or the mods in repos owned by the AzerothCore GitHub users? If so:)

I give full consent for the relicensing of all my past contributions to AzerothCore to GPLv2+

**Comment by Xtian2108** [comment]
I give full consent for the Relicensing of all my contributions to Azerothcore

**Comment by elthehablo** [comment]
I give full consent for the relicensing of all my past contributions to AzerothCore to GPLv2+

**Comment by Efymer** [comment]
I give full consent for the relicensing of all my past contributions to AzerothCore to GPLv2+

**Comment by sudlud** [comment]
I give full consent for the Relicensing of all my contributions to Azerothcore

**Comment by Pyromanihack** [comment]
I give full consent for the Relicensing of all my contributions to Azerothcore

**Comment by neifion-00000000** [comment]
I give full consent for the Relicensing of all my contributions to Azerothcore

**Comment by MikaMauger** [comment]
I give full consent for the relicensing of all my contributions to AzerothCore.

**Comment by FallenDev** [comment]
I give full consent

**Comment by FrancescoBorzi** [comment]
I give full consent for the relicensing of all my contributions to AzerothCore.

**Comment by walkline** [comment]
I give full consent for the relicensing of all my past contributions to AzerothCore to GPLv2+

**Comment by ReynoldsCahoon** [comment]
I give full consent for the relicensing of all my past contributions to AzerothCore to GPLv2+

**Comment by Alyst3r** [comment]
One not too huge contrib but regardless, I give full consent for the relicensing of all my contributions to Azerothcore

**Comment by Azcobu** [comment]
I give full consent for the relicensing of all my past contributions to AzerothCore to GPLv2+.

**Comment by AnchyDev** [comment]
I hereby give my full consent for the re-licensing of all my past, present, and potentially mystical future contributions to GPLv2+ for the AzerothCore project, on the condition that I receive a pet murloc, and the title 'Keeper of Code' in every future commit log. 

Failure to meet these terms may result in minor disappointment, but no actual consequences.

<sup><sub>The above consent is indeed real and not a joke. You may proceed with the relicensing as specified.</sub></sup>

**Comment by ghost** [comment]
I give full consent for the relicensing of all my past contributions to AzerothCore to GPLv2+

**Comment by Kaev** [comment]
I give full consent for the relicensing of all my contributions to Azerothcore.

**Comment by ghost** [comment]
I give full consent for the relicensing of all my past contributions to AzerothCore to GPLv2+. This also includes my module contributions and the module "mod-morphsummon" (https://github.com/azerothcore/mod-morphsummon). The account used for the commits was `38475780+Stoabrogga@users.noreply.github.com`.
After the relicensing is finished I'll also relicense my old project, here's the GitHub mirror:
https://github.com/opfesoft/sol
I spent 4 years working on this, take from it what you want, I'll not touch 335 anymore.

**Comment by ljfp** [comment]
I give full consent for the Relicensing of all my contributions to Azerothcore.

**Comment by Tralenor** [comment]
I give consent for my contributions to the azerothcore-wotlk repository to be relicensed from AGPLv3 to GPLv2+

**Comment by pangolp** [comment]
I give full consent for the relicensing of all my past contributions to AzerothCore to GPLv2+

**Comment by aradep** [comment]
I give full consent for the relicensing of all my contributions to AzerothCore.

**Comment by ghost** [comment]
I give full consent for the relicensing of all my contributions to Azerothcore.

I've asked also to Maelthyrr, an old AC staff, who has deleted their account and can therefore no longer reply. He give also this full consent for the relicensing of all these contributions to Azerothcore.

**Comment by Hacki95** [comment]
I give full consent for the Relicensing of all my contributions to Azerothcore

**Comment by peti446** [comment]
I give full consent for the relicensing of all my past contributions to AzerothCore to GPLv2+

**Comment by 3ster** [comment]
I give full consent for the relicensing of all my past contributions to AzerothCore to GPLv2+

**Comment by sogladev** [comment]
I give consent for the relicensing of all my past contributions to AzerothCore to GPLv2+

**Comment by MichalVytrhlik** [comment]
I give full consent for the Relicensing of all my contributions to Azerothcore

**Comment by trinitarius** [comment]
I give full consent for the relicensing of all my past contributions to AzerothCore to GPLv2+

**Comment by podmepodme** [comment]
I give full consent for the Relicensing of all my contributions to Azerothcore

**Comment by Kitzunu** [comment]
I give full consent for the relicensing of all my past contributions to AzerothCore to GPLv2+

**Comment by kamenitxan** [comment]
I give full consent for the relicensing of all my past contributions to AzerothCore to GPLv2+

**Comment by kjack9** [comment]
I give full consent for the relicensing of all my past contributions to AzerothCore to GPLv2+

I additionally give full consent for the relicensing of all of my work on the `mod-autobalance` module, located at https://github.com/azerothcore/mod-autobalance

**Comment by Nefertumm** [comment]
I fully consent to relicensing all my past contributions to AzerothCore and their modules (including my own) to GPLv2+.

**Comment by ZhengPeiRu21** [comment]
I consent to relicensing my past contributions to AzerothCore and their modules to GPLv2+.

**Comment by vurtne** [comment]
我完全同意重新授权我对 Azerothcore 的所有贡献

**Comment by ianmeesters** [comment]
I give full consent for the Relicensing of all my contributions to Azerothcore

**Comment by Annamaria-CC** [comment]
I give full consent for the relicensing of all my few contributions I did to Azerothcore to GPLv2

**Comment by PkllonG** [comment]
I consent to relicensing my past contributions to AzerothCore and their modules to GPLv2+.

**Comment by blinkysc** [comment]
I give full consent for the Relicensing of all my contributions to Azerothcore

**Comment by nolt** [comment]
I give full consent for the relicensing of all my past contributions to AzerothCore to GPLv2+
